jsettlers.graphics/src/main/resources/jsettlers/graphics/localization/labels_en.properties

```diff
@@ -198,8 +198,13 @@ attacked    = You have been attacked.
 minerals_COAL = A geologist found coal.
 minerals_IRONORE = A geologist found iron.
 minerals_GOLDORE = A geologist found gold.
-cannot_find_work_LUMBERJACK = Your woodcutter can't find any trees in his work area.
-cannot_find_work_STONECUTTER = Your Stonecutter can't find any stone in his work area.
+cannot_find_work_LUMBERJACK = Your woodcutter can't find any trees in its work area.
+cannot_find_work_STONECUTTER = Your stonecutter can't find any stone in its work area.
+cannot_find_work_FORESTER = Your forester can't find any free space for new trees in its work area.
+cannot_find_work_FARM = Your grain farm can't find any free space for new trees in its work area.
+cannot_find_work_WINEGROWER = Your winery can't find enough spots for growing vines in its work area.
```

Comment:
This message is not only shown if it cannot find any work, but also if it has to few spots to work all the time. If the farmer or winegrower is blocked due to a limitted amount of positions, it will notify the user as well.